### DISCUSSION TITLE
几个功能建议

### DISCUSSION THREAD

**Discussion by MIONKING** [the1812/Bilibili-Evolved#2243] [功能建议 / Ideas]
1.现有弹幕的点赞功能不能用了
2.可以加个自动加载视频的功能，有时候网络环境不是很稳定，网速时快时慢，可以让他一直加载1/2或者1/3这样选项的量，也可以在还没播放的加载点之前就提前去加载下一段的缓冲，可以提高视频观看的流畅性。
3.可以加一些快捷键，比如在HTML5视频增强的脚本里面的，Z还原播放速度，X-0.1倍速度，C+0.1倍数，这种功能对学生狗来说简直太需要了

谢谢大佬的脚本，超级好用~

**Comment by Geeyun-JY3** [comment]
> 1.现有弹幕的点赞功能不能用了

见：https://github.com/the1812/Bilibili-Evolved#安装
> 做好觉悟, 脚本开启后不能使用弹幕点赞和举报,

> 3.可以加一些快捷键，比如在HTML5视频增强的脚本里面的，Z还原播放速度，X-0.1倍速度，C+0.1倍数，这种功能对学生狗来说简直太需要了

有这功能：`快捷键扩展`。如果有自定义播放速度需求再开 `扩展视频倍速菜单` 后自己输入所需倍数
